README.adoc

```diff
@@ -361,7 +362,8 @@ The [hotspot=inventory file=1]`inventory` application is configured similarly to
 and additionally includes the [hotspot=service file=1]`service` and [hotspot=expose file=1]`expose` parameters. The
 [hotspot=servicePort file=1]`service.port` parameter specifies the port exposed by the container, allowing the application
 to be accessed from outside the container. To access the application from outside of the cluster, it needs to be exposed,
-which is done by setting the [hotspot=expose file=1]`expose` parameter to `true`.
+which is done by setting the [hotspot=expose file=1]`expose` parameter to `true`, which will let the operator create and
```

Comment:
It might be good to break this into two sentences to avoid two `which`

Reply:
Thanks, done!